src/main/native/jgskit.win64.cygwin.mak

```diff
@@ -0,0 +1,95 @@
+###############################################################################
+#
+# Copyright IBM Corp. 2023
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution.
+#
+###############################################################################
+
+TOPDIR = $(MAKEDIR)\..\..\..
+
+PLAT = win
+BUILDTOP = $(TOPDIR)\target\build$(PLAT)
+HOSTOUT = $(BUILDTOP)\host64
+JAVACLASSDIR = $(TOPDIR)\target\classes
+#JCE_CLASSPATH ?= $(JAVACLASSDIR)/../ibmjceplus.jar:../../../target/misc.jar
```

Comment:
Can this commented out line be deleted. If it is not needed it can be removed.

Reply:
Deleted.